prototypes/Eclipse GUI Prototype/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Probably the error occurs for you because you did not re-import the project after the renaming in a16fc94